models/maat/R/model2netcdf.MAAT.R

```diff
@@ -50,6 +31,37 @@ model2netcdf.MAAT <- function(rundir, outdir, sitelat = -999, sitelon = -999, st
   # setup constants
   day_secs <- udunits2::ud.convert(1, "day", "seconds")
   
+  # setup helper function
+  var_update <- function(data, out, oldname, newname, oldunits, newunits=NULL, missval=-999, longname, ncdims) {
+    # ifelse is no longer working as expected, so now we have this function to deal with any Inf values
+    f_sort <- function(s) {
+      if (is.infinite(s)) -999
```

Comment:
I would consider the following instead. `is.finite(x)` is `FALSE` for a wider variety of `x` than just infinity (e.g. it will also pick up `NA` and `NaN`). 
```suggestion
      if (!is.finite(s)) -999
```

Reply:
I need to test this first, just to be sure.